samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml

```
@@ -5,8 +5,50 @@ metadata:
   name: buildah
 spec:
   buildSteps:
+    - name: prepare
+      image: docker.io/library/bash:5.1.4
+      imagePullPolicy: Always
+      securityContext:
+        privileged: true
+      command:
+        - /usr/local/bin/bash
+      args:
+        - -c
+        - |
+          set -eo pipefail
+
+          echo "> Creating registries config file..."
+
+          formate(){
+              array=(`echo $1 | tr ',' ' '`)
+              str=""
+              for m in "${array[@]}"; do
+                  str=$str"'${m}', "
+              done
+
+              echo ${str%??}
+          }
+
+          path="$(params.shp-source-context)/registries.conf"
```

I think we should not add any files to the user's source. We could add an empty dir, but I suggest to write to /tmp and to move the whole new script to the build-and-push step. I do not see a reason to have another container.

```suggestion
          path="/tmp/registries.conf"
```

---

Done